tests/test_benchmark_stratified_thermal_storage.py

```diff
@@ -0,0 +1,836 @@
+                    )
+                except:
+                    print(
+                        "Please check the main input parameters for errors. "
+                        "This exception prevents that D1_model_components.py is "
+                        "overwritten in case running the main errors out."
+                    )
+
+                # Revert changes made in D1_model_components.py
+                mvs_D1_modified = open(self.D1, "w")
+                mvs_D1_modified.write(mvs_D1)
+                mvs_D1_modified.close()
+
+                results_generic_storage = pd.read_excel(
+                    os.path.join(
+                        TEST_OUTPUT_PATH, use_case, "timeseries_all_busses.xlsx"
+                    ),
+                    sheet_name="Heat",
+                )
+
+            elif use_case == "Stratified_thermal_storage_optimize":
+                main(
+                    display_output="warning",
+                    path_input_folder=TEST_INPUT_PATH,
+                    path_output_folder=os.path.join(TEST_OUTPUT_PATH, use_case),
+                    input_type="csv",
+                    overwrite=True,
+                    save_png=False,
+                    lp_file_output=True,
+                )
+
+                results_stratified_thermal_storage = pd.read_excel(
+                    os.path.join(
+                        TEST_OUTPUT_PATH, use_case, "timeseries_all_busses.xlsx"
+                    ),
+                    sheet_name="Heat",
+                )
+
+        assert (
+            results_generic_storage["TES input power"].values.all()
+            == results_stratified_thermal_storage["TES input power"].values.all()
+        ), f"When the parameters {THERM_LOSSES_REL} and {THERM_LOSSES_ABS} are commented out in {self.D1} the results of the simulation should be the same as if {THERM_LOSSES_REL} and {THERM_LOSSES_ABS} are not used in the simulation"
+        storage_data_original.to_csv(self.storage_csv)
+
+    def teardown_method(self):
+        use_cases = [
+            "Generic_storage_fix",
+            "Stratified_thermal_storage_fix",
+            "Generic_storage_optimize",
+            "Stratified_thermal_storage_optimize",
+        ]
+        for use_case in use_cases:
+            if os.path.exists(os.path.join(TEST_OUTPUT_PATH, use_case)):
+                shutil.rmtree(
+                    os.path.join(TEST_OUTPUT_PATH, use_case), ignore_errors=True
+                )
+
+    # # this ensure that the test is only ran if explicitly executed, ie not when the `pytest` command
+    # # alone is called
+    @pytest.mark.skipif(
+        EXECUTE_TESTS_ON not in (TESTS_ON_MASTER),
+        reason="Benchmark test deactivated, set env variable "
+        "EXECUTE_TESTS_ON to 'master' to run this test",
+    )
+    @mock.patch("argparse.ArgumentParser.parse_args", return_value=argparse.Namespace())
+    def test_fixed_losses_higher_invested_storage_capacity_float(self, margs):
```

Comment:
Could you add a short explanation of what you test and what you expect to the remaining benchmark tests, as well? Can be very short in this case.. if there are no losses you need less capacity..

Reply:
Absolutely! I must have forgotten it, thank you.
Done with commit https://github.com/rl-institut/multi-vector-simulator/pull/718/commits/f185e26133f1b435d86c1d899d1c60e136be0fec